frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -93,6 +94,7 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 					/>
 				))}
 			</List>
+			<GroupFiltersWithTriggerButton />
```

Comment:
Can you rename this to `<GroupRules />` ?
Treat this as an Input, add `value` , 'onChange' props
That way when we use it in the group form is just another input that changes the rules of the group

Reply:
```
const handleOnChange = (value) => {
      group.rules = value;
}

<GroupRules  value={group.rules}  onChange={handleOnChange} />
```